docker-compose.yml

```diff
@@ -52,6 +58,10 @@ mysql-data:
 
 mysql:
   build: ./docker/images/mysql
+  entrypoint:
+    - docker-entrypoint.sh
```

Comment:
So I tested this on a different machine, and it worked without the change I suggested above. After further research I found out that the mysql:5.6 image had been cached on my laptop from 14 months ago, and the newer version with the same tag had a new file, `/usr/local/bin/docker-entrypoint.sh`, which was not present in the older version. The `/entrypoint.sh` file is still there in the newer version and there was no difference between the two files. Note that I had run `docker-compose pull` before posting this comment, but that does *not* update recursively. This is an edge case that most developers won't run into, but the change wouldn't hurt.